src/auditwheel/repair.py

```diff
@@ -30,6 +30,18 @@
 ).match
 
 
+def _is_in_list(soname: str, items: Optional[List[str]]) -> Optional[str]:
+    for item in (items or []):
+        if item in soname:
+            return item
+
+    return None
+
+
+def _filter(items: Optional[List[str]]) -> List[str]:
```

Comment:
It's a one liner used exactly once, having a function does not help readability and/or factorization here.